deploy/olm-catalog/nightly/eclipse-che-preview-openshift/csv-config.yaml

```diff
@@ -1,3 +1,3 @@
 operator-path: deploy/operator.yaml
 role-paths: [ "deploy/olm-catalog/nightly/eclipse-che-preview-openshift/generated/roles/role.yaml", "deploy/olm-catalog/nightly/eclipse-che-preview-openshift/generated/roles/cluster_role.yaml"]
-crd-cr-paths: ["deploy/crds/org_v1_che_crd.yaml"]
+crd-cr-paths: ["deploy/crds/org_v1_che_crd.yaml", "deploy/crds/org.eclipse.che_checlusterbackups_crd.yaml", "deploy/crds/org.eclipse.che_checlusterrestores_crd.yaml"]
```

Comment:
There is one more file to update
https://github.com/eclipse-che/che-operator/blob/main/deploy/olm-catalog/nightly/eclipse-che-preview-kubernetes/csv-config.yaml